lib/vsc/utils/mail.py

```diff
@@ -68,8 +67,45 @@ def __init__(self, mail_host=None, mail_to=None, mail_from=None, mail_subject=No
 class VscMail(object):
     """Class providing functionality to send out mail."""
 
-    def __init__(self, mail_host=None):
-        self.mail_host = mail_host
+    def __init__(self, mail_host=None, mail_port=None, smtp_auth_user=None, smtp_auth_password=None, smtp_use_starttls=False):
+
+        # we must not have the port in the host as this will fail the certificate check if needed
+        (host, port) = (mail_host, mail_port)
+        if mail_host:
+            try:
+                (host, port_) = mail_host.split(":")
```

Comment:
why is the even allowed?

Reply:
why is the what?

Reply:
the format with a `:` in it

Reply:
Dunno, somebody passed `explout.ugent.be:587` to it at some point :p

Reply:
ah no, my bad, the config has `set smtp=exploud.ugent.be:587`